ydb/core/tx/schemeshard/schemeshard_info_types.h

```diff
@@ -2371,11 +2371,16 @@ struct TTableIndexInfo : public TSimpleRefCount<TTableIndexInfo> {
     using EType = NKikimrSchemeOp::EIndexType;
     using EState = NKikimrSchemeOp::EIndexState;
 
-    TTableIndexInfo(ui64 version, EType type, EState state)
+    TTableIndexInfo(ui64 version, EType type, EState state, std::string_view description)
```

Comment:
Is there a reason to use `std::string_view` here?

Reply:
In my head it's commonly best way to pass something like const string reference.
Also `ParseFromString` uses `string_view` (abseil but it's kind of same):
```cpp
bool ParseFromString(y_absl::string_view data);
```

Do you have some other preference? What and why?

Reply:
`const TString&`.
We don't store original argument value in the object and don't construct TTableIndexInfo from anything but TString value, so string_view doesn't bring any benefits here.